hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3MultipartUploadCompleteRequest.java

```diff
@@ -126,7 +128,9 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       multipartKey = omMetadataManager.getMultipartKey(volumeName,
           bucketName, keyName, uploadID);
 
-      // TODO to support S3 ACL later.
+      // check Acl
+      checkKeyAcls(ozoneManager, volumeName, bucketName, keyName,
+          IAccessAuthorizer.ACLType.WRITE, OzoneObj.ResourceType.KEY);
```

Comment:
Should we check for CREATE as we are completing the key write by adding new entry in the KeyTable?

Reply:
Xiaoyu, from user's point of view,  to upload a object, both CREATE and WRITE permission are needed.  Currently OmKeyCreateRequest checks CREATE permission and OmKeyCommitRequest checks WRITE permission.  Multi-upload follows the same rule,  CREATE permission is checked when initiating a multi-upload request.

Reply:
Sounds good tome. +1.